README.md

```diff
@@ -240,6 +240,13 @@ dialog.off('show', doSomething);
 
 Nested dialogs is a [questionable design pattern](https://ux.stackexchange.com/questions/52042/is-it-acceptable-to-open-a-modal-popup-on-top-of-another-modal-popup) that is not referenced anywhere in the [HTML 5.2 Dialog specification](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element). Therefore it is discouraged and not supported by default by the library. That being said, if you still want to run with it, [Renato de Leão explains how in edenspiekermann/a11y-dialog#80](https://github.com/edenspiekermann/a11y-dialog/issues/80#issuecomment-377691629).
 
+## Implementations
+
+If you happen to work with [React.js](https://github.com/facebook/react/) or [Vue.js](https://github.com/vuejs/vue) in your project, you're lucky! There are already great light-weight wrapper implemented for `a11y-dialog`:
```

Comment:
1. Maybe we use “React”/“Vue” rather than “React.js”/“Vue.js”? 
2. Can we use curly quotes/apostrophes? :)